### ISSUE TITLE
Not able to access notes attached with a node

### ISSUE THREAD

**Issue by kraghuprasad** [nnako/freeplane-python-io#6]
I was going through the document to find out how to get the contents of a note attached with a node. But I could not find any details on how to do it, even though the word note and notes are mentioned many times in the API document. Has anyone used freeplane-python-io library to retrieve such contents along with all list of linked (incoming and outgoing) nodes?

Thanks in advance.

**Comment by nnako** [comment]
Hi @kraghuprasad ,

thank you for opening this issue.

Unfortunately, the `notes` attribute of nodes is not fully implemented, yet. But, as it goes along with the `details` attributes, you could use those details for storing information, if you have not already done so in parallel to your notes.

With the next release, the handling of `notes` attributes will be covered within the codebase. At least to handle `plaintext` notes. I guess, already until the end of the current week. HTML handling within notes would then be implemented later.

Thanks.
Nnako

**Comment by nnako** [comment]
Hi @kraghuprasad ,

I just pushed the added feature. Now, you would be able to use it in the following way:

- `<mindmap>.find_nodes( notes="<text>" )`
to get a list of `<mindmap>`'s nodes which contain the `<text>` within their nodes attribute

- `<node>.find_nodes( notes="<text>" )`
to get a list of nodes below the tree starting with `<node>` which contain the `<text>` within their nodes attribute

- `<node>.find_children( notes="<text>" )`
to get a list of `<node>`'s direct child nodes which contain the `<text>` within their nodes attribute

- `<node>.notes = "<text>"`
set the `<node>`'s notes attribute to a specific plaintext value

- `<var> = <node>.notes`
set the `<var>` variable / object with the plaintext string value of the `<node>`'s notes attribute

Does this work for you?

Thanks.

**Comment by nnako** [comment]
Now version 0.9.0 can be updated directly via pip. It should solve the issue. Please confirm.

**Comment by kraghuprasad** [comment]
Thanks a lot, nnako, for the prompt replies and the fixes. :)

I confirm that I am able to access node.notes successfully. I would be testing other methods soon.

Thanks again.

**Comment by nnako** [comment]
closing issue as it seems completed.
